packages/terra-table/src/Table.jsx

```diff
@@ -169,6 +169,11 @@ const propTypes = {
    * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+  /**
+   * @private
+   * True if Table is invoked from WorkListDataGrid component
+   */
+  fromWorkListDataGrid: PropTypes.bool,
```

Comment:
We should not have a reference to the WorklistDataGrid in the Terra Table component.  The Terra Table is the base component.  We would not add a property mentioning a specific implementation.

Reply:
Updated the prop name to rowMinimumHeight.
commit: 5cbf782d540fe32fe513e317693cf9092bd902eb
Thanks